README.md

```diff
@@ -0,0 +1,20 @@
+# README
+
+## Blacklight Discovery
+
+* Ruby version 2.5.7p206
+
+* Rails version 5.1.7
+
+* Blacklight 7.4.1
+
+## Running locally
+
+1. Clone the git repo: `git@github.com:emory-libraries/blacklight-catalog.git`
```

Comment:
```suggestion
1. Clone the git repo: `git clone git@github.com:emory-libraries/blacklight-catalog.git`
```

Reply:
That has been changed in the last commit.